src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -593,12 +598,14 @@ type FsharpLspServer(commands: Commands, lspClient: FSharpLspClient) =
                         DocumentFormattingProvider = Some true
                         DocumentRangeFormattingProvider = Some false
                         SignatureHelpProvider = Some {
-                            SignatureHelpOptions.TriggerCharacters = Some [| "("; ","|]
+                            TriggerCharacters = Some [| "("; ","; "<"; " "; |]
```

Comment:
I think I need to revert this and include it as part of the signature help work I'm doing in another branch.

Reply:
the most recent commit has reverted this